apps/edxapp/templates/_volumes/export.yml.j2

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: edxapp-pvc-export
+  namespace: "{{ project_name }}"
+  labels:
+    app: edxapp
+    version: "{{ edxapp_image_tag }}"
+    deployment_stamp: "{{ deployment_stamp }}"
+spec:
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: {{ edxapp_export_volume_size }}
```

Comment:
You should also remove the `exports.yml.j2` volume.

Reply:
👍